src/Umbraco.Headless.Client.Net/Management/FormService.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Refit;
+using Umbraco.Headless.Client.Net.Configuration;
+using Umbraco.Headless.Client.Net.Management.Models;
+
+namespace Umbraco.Headless.Client.Net.Management
+{
+    internal class FormService : IFormService
```

Comment:
Any particular reason why its called `FormService` and not just `Forms` ?

Reply:
The other classes in management is also called `Service`

Reply:
yes, sorry, I confused myself. This naming around Forms vs FormService was mostly around the property name in `ContentManagementService`, which you fixed already.